pmdarima/arima/seasonality.py

```diff
@@ -226,12 +228,15 @@ def _sd_test(wts, s):
 
         # no use checking, because this is an internal method
         # if n <= s:  raise ValueError('too few samples (%i<=%i)' % (n, s))
-        frec = np.ones(int((s + 1) / 2), dtype=np.int)
+        frec = np.ones(int((s + 1) / 2), dtype=int)
         ltrunc = int(np.round(s * ((n / 100.0) ** 0.25)))
         R1 = CHTest._seas_dummy(wts, s)
 
         # fit model, get residuals
-        lmch = LinearRegression(normalize=True).fit(R1, wts)
+        lmch = make_pipeline(
+            StandardScaler(with_mean=False),
+            LinearRegression()
+        ).fit(R1, wts)
```

Comment:
I can't really remember everything that leads up to this call, but are we assuming data is already centered? (e.g., `with_mean=False`)